ui/pages/onboarding-flow/metametrics/__snapshots__/metametrics.test.js.snap

```diff
@@ -36,7 +36,7 @@ exports[`Onboarding Metametrics Component should match snapshot 1`] = `
     <ul>
       <li>
         <div
-          class="box box--flex-direction-row"
+          class="mm-box"
```

Comment:
The old `Box` would automatically add the flex-direction-row class but still default to display block. The new `Box` doesn't have flex direction by default, which is why the class isn't there. The elements are horizontally aligned because they are all `span`s. This was a bug with the old `Box`. This CSS wouldn't do anything until you added flex props.

<img width="388" alt="Screenshot 2024-07-29 at 4 03 35 PM" src="https://github.com/user-attachments/assets/a13bdcf0-1933-44d6-819c-1c437fe1edb8">